api/lib/handlers/project/give/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var ApiService = require('../../../ApiService.js').ApiService;
+var error = require('../../../config.js').errors;
+var async = require('async');
+
+module.exports.post = function(req, res, next) {
+  const id = req.params.id;
+  const amount = req.body.amount;
+  const userId = req.body.userId;
+
+  async.waterfall([
+    function checkArguments(cb) {
+      if (amount <= 0)
+        return cb(error['AMOUNT_INVALID'], null)
+      return cb(null);
+    },
+    function checkUser(cb) {
+      const result = ApiService('SELECT * FROM give_me_time_public.person WHERE id=($1)',
+      [userId],
+      (err, result) => {
+        if (!err && !result.id)
+          err = error['UNKNOWN_PERSON']
```

Comment:
you could use this syntax instead to avoid magic strings: `error.UNKNOWN_PERSON`